ush/python/pygfs/task/atmens_analysis.py

```diff
@@ -234,8 +233,20 @@ def get_bkg_dict(self, task_config: Dict[str, Any]) -> Dict[str, List[str]]:
             # accumulate directory list for member restart files
             dirlist.append(os.path.join(task_config.DATA, 'bkg', memchar))
 
-            # get FV3 RESTART files, this will be a lot simpler when using history files
-            rst_dir = os.path.join(task_config.comin_ges_atmens, memchar, 'atmos/RESTART')
+            # get FV3 restart files, this will be a lot simpler when using history files
+            template = task_config.COM_ATMOS_RESTART_TMPL
+            tmpl_dict = {
+                'ROTDIR': task_config.ROTDIR,
+                'RUN': self.runtime_config.RUN,
+                'YMD': to_YMD(task_config.previous_cycle),
+                'HH': task_config.previous_cycle.strftime('%H')
+            }
```

Comment:
This doesn't change from member to member, so can be moved before the loop.

Reply:
This code is now in `ush/python/pygfs/task/analysis.py`.   Requested change made at [`37fbac4`](https://github.com/RussTreadon-NOAA/global-workflow/commit/37fbac4bb01b74a9a642d2fbb3a871f2aafc6432)